### ISSUE TITLE
Latest release breaks when Elementor is installed

### ISSUE THREAD

**Issue by bompus** [bonny/WordPress-Simple-History#151]
When trying to view almost any page in WP admin, I get this error:

Uncaught Error: Class 'Elementor\Control_Wysiwyg' not found in /******/wp-content/plugins/elementor/includes/managers/controls.php:415

This is probably due to trying to load the logger everywhere?

**Comment by bonny** [comment]
Thanks for the bug report. I will install Elementor and see if I can reproduce.

**Comment by bonny** [comment]
Is is this one btw?
https://sv.wordpress.org/plugins/elementor/

**Comment by bompus** [comment]
Yes. I am using the Pro version, but the free should work as well. Strangely, when I delete your plugin, I still get the error, but everything was fine until I updated your plugin to latest, then it all went haywire. Trying to track it down and get my site back up now :)

**Comment by bonny** [comment]
No other plugins (or anything else) updated recently too?

**Comment by bompus** [comment]
Logged in this morning and simple history was the only plugin that needed an update, and that is all I updated. Site was working fine before then.

Just got the site working again. To fix, I had to:

1. Delete simple history folder from plugins
2. Rename elementor directory to elementor-disabled
3. Clear PHP OPCache
4. Rename elementor-disabled back to elementor
5. Activate elementor

If you aren't running PHP with opcache, you probably won't have the problem after deleting the simple history folder, but if others experience the same issue, the above steps worked for me.
